stonesoup/simulator/tests/conftest.py

```diff
@@ -30,3 +36,52 @@ def function(state, noise):
             matrix = np.array([[1, 0, 0, 0], [0, 0, 1, 0]])
             return matrix @ state.state_vector
     return TestMeasurementModel()
+
+
+@pytest.fixture()
+def sensor_model1():
+    class TestSensor():
+
+        @staticmethod
+        def measure(ground_truth):
+            return Detection(ground_truth.state_vector[(0, 2), :],
+                             timestamp=ground_truth.timestamp)
+    return TestSensor()
+
+
+@pytest.fixture()
+def sensor_model2():
+    class TestSensor:
+
+        @staticmethod
+        def measure(ground_truth):
+            return Detection(ground_truth.state_vector,
+                             timestamp=ground_truth.timestamp)
+    return TestSensor()
+
+
+@pytest.fixture()
+def platform_class():
```

Comment:
Is there a need to use a custom platform class rather than one of the existing ones? There may well be, but can you explain what value it adds?

Reply:
The test platform and sensor classes make detections predictable and hence easier to test.